src/pages/home/report/ReportActionItemParentAction.tsx

```diff
@@ -31,9 +25,15 @@ type ReportActionItemParentActionProps = ReportActionItemParentActionOnyxProps &
     /** The id of the report */
     // eslint-disable-next-line react/no-unused-prop-types
     reportID: string;
+
+    /** The current report is displayed */
+    report: OnyxEntry<OnyxTypes.Report>;
+
+    /** Report actions belonging to the report's parent */
```

Comment:
@kacper-mikolajczak can you fix this in some future PR please?
```suggestion
    /** Report action belonging to the report's parent */
```